app/model.py

```diff
@@ -1001,35 +1012,52 @@ def create_id():
         return unique_id.key().id()
 
 class UsageCounter(db.Model):
-    """Count the history usage for each repo"""
+    """Counters count the historical statistics for a given repository.
+    To see how this is used, check out admin_stastics.py.
+    Unlike the Counter class, UsageCounter object increment when
+    a new record or a new note is created, which means, the UsageCounter
+    will not decrement when a record/note is expired/deleted."""
+
+    # repo stored as a seperate property so it can be indexed and queried.
     repo = db.StringProperty(required=True)
+    # person_counter increment when a new person record is created.
     person_counter = db.IntegerProperty(default=0, required=True)
+    # note_counter increment when a new note is added in a person's record.
     note_counter = db.IntegerProperty(default=0, required=True)
 
     @staticmethod
-    def create(repo, person_counter=0, note_counter=0):
-        """Create a new counter"""
-        return UsageCounter(key_name=repo, repo=repo,
-            person_counter=person_counter, note_counter=note_counter)
+    def get(repo):
+        """Gets the entity with a given repository."""
+        return UsageCounter.get_by_key_name('%s' % repo)
```

Comment:
Still not simplified :) ```'%s' % repo``` -> Just ```repo```

```'%s' % x``` is meaningless if x is already a string. BTW Even if x is not a string, you should write str(x) instead.